spec/factories/vet360/emails.rb

```diff
@@ -4,7 +4,7 @@
   factory :email, class: 'Vet360::Models::Email' do
     sequence(:id) { |n| n }
     sequence(:email_address, 100) { |n| "person#{n}@example.com" }
-    sequence(:transaction_id, 100) { |n| "#{n}" }
+    sequence(:transaction_id, 100, &:to_s)
```

Comment:
I would make sure these transaction_id's are generated as UUID's so it's consistent with what we are getting from Vet360